easybuild/easyconfigs/f/FIX/FIX-1.068-foss-2019a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'FIX'
+version = '1.068'
+
+homepage = 'https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX'
+description = """FIX attempts to auto-classify ICA components into "good" vs "bad" components, so that the bad
+components can be removed from the 4D FMRI data."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://www.fmrib.ox.ac.uk/~steve/ftp/']
+sources = ['fix.tar.gz']
+checksums = ['bb10a24f28b3c26a382744faa127c1c7bf3167dc7a4f06da1aab89b58b74ac72']
+
+dependencies = [
```

Comment:
@crubb please add Python dependency, and add Python versionsuffix (below version)?

Reply:
`FIX` does not depend on Python, but `VTK` down the line depends on Python.

Reply:
@crubb if `VTK` depends on Python, then `FIX` also (indirectly) depends on Python ;)
the reason you have to add it is so you can use the `%(pyver)s` template in the versionsuffix.